reporters/GlobalSummaryReporter.js

```diff
@@ -141,13 +161,15 @@ class GlobalSummaryReporter {
       // If we were using `endBlockNumberForPeriod` to query data for some other pair as well,
       // then we wouldn't be able to set `endBlockNumberForPeriod = latestSwapBlockNumber`
       this.uniswapClient = getUniswapClient();
-      const latestSwap = (await this.uniswapClient.request(queries.LAST_TRADE_FOR_PAIR(this.uniswapPairAddress)))
+      const latestSwap = (await this.uniswapClient.request(uniswapQueries.LAST_TRADE_FOR_PAIR(this.uniswapPairAddress)))
         .swaps[0].transaction;
       this.latestSwapTimestamp = latestSwap.timestamp;
       this.latestSwapBlockNumber = Number(latestSwap.blockNumber);
-      // Note: `endBlockNumberForPeriod` is the highest block number that we will manually query for.
+      // Note: `lastExchangeBlockNumberForPeriod` is the highest block number that we will manually query the uniswap subgraph for.
       if (this.endBlockNumberForPeriod > this.latestSwapBlockNumber) {
-        this.endBlockNumberForPeriod = this.latestSwapBlockNumber;
+        this.lastExchangeBlockNumberForPeriod = this.latestSwapBlockNumber;
```

Comment:
This is outside the scope of this PR but I want to explain this important fix.

We don't want to query the uniswap subgraph for a block that is > the block # of the latest swap. However, we also don't want to set `this.endBlockNumberForPeriod` to the `latestSwapBlockNumber`. Imagine that the latest swap # is 1 day ago, then that means that the `this.endBlockNumber` will also be 1 day ago, but this will prevent us from querying "period" data for non-uniswap data correctly. 

For example, take ETHBTC: the latest uniswap swap was 1 day ago, so we'll only query uniswap data up until then for the "period". However, when querying uma event data we still want the period to end at the current block.